tests/_cli/test_markdown_conversion.py

```diff
@@ -193,7 +193,11 @@ def test_markdown_with_sql() -> None:
 
     # SQL notebook
 
-    ```sql {.marimo query="export"}
+    ```python {.marimo}
+    mem_engine = mo.create_engine("sqlite:///:memory:")
```

Comment:
nit: could we get rid of `mo.create_engine` in favor of `create_engine` in case people think its a real function when looking through the codebase